### DISCUSSION TITLE
Welcome to Anthropic Discussions!

### DISCUSSION THREAD

**Discussion by kayhantolga** [betalgo/Anthropic#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by msheffwaldrop** [comment]
Does anyone know a way to get responses from Anthropic. I've sent 2 support emails with no response in over 48 hours. I can't log in by web interface any longer.
